tests/backend/test_twomode_squeezing_operation.py

```diff
@@ -0,0 +1,70 @@
+# Copyright 2019 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+r"""
+Unit tests for two-mode squeezing operation
+Convention: The squeezing unitary is fixed to be
+U(z) = \exp(0.5 (z^* \hat{a}_1\hat{a}_2 - z (\hat{a^\dagger}_1\hat{a}_2)))
+where \hat{a} is the photon annihilation operator.
+"""
+# pylint: disable=too-many-arguments
+import itertools as it
+
+import pytest
+import numpy as np
+
+
+PHASE = np.linspace(0, 2 * np.pi, 3, endpoint=False)
+MAG = np.linspace(0.0, 0.2, 5, endpoint=False)
+
+
+def get_amplitude(k, r, p):
+    """ Get amplitude for two mode squeezing operation"""
+    if k[0] == k[1]:
+        tmsv = (np.exp(1j * p) * np.tanh(r)) ** k[0] / np.cosh(r)
+        return tmsv
+    return 0.0
```

Comment:
Nice!

Reply:
It might be worth adding to the docstring a description of the arguments and return type

Reply:
Added docstrings with args and return